snuba/util.py

```diff
@@ -150,21 +150,8 @@ def column_expr(dataset, column_name, body, alias=None, aggregate=None):
         return complex_column_expr(dataset, column_name, body)
```

Comment:
Are you planning to deal with this case in a separate PR ?

Reply:
Were you thinking this should also move to events? I assumed that this could stay as-is since we'd support the same syntax for complex/nested expressions across various datasets

Reply:
Sorry, my bad, I was thinking about an older version. You are right, this should not move